fastlane/docs/Platforms.md

```diff
@@ -1,6 +1,6 @@
 # Support for multiple platforms
 
-`fastlane` can handle multiple app platforms in one `Fastfile`. This is super useful for projects supporting iPhone, macOS and Android apps.
```

Comment:
As I'm playing a lot with platforms in Fastlane and my main intention here was fix the iPhone/iOS issue.

While doing the change, I read the sentence and the "and" looked to me a bit odd because gave me the impression that all criterias should be met (iOS && macOS && Android), of course that is not a big issue and we can keep "and" if you think it's more suitable.